### PR TITLE
fix: update prettier config

### PR DESCRIPTION
**Pull request by frytg**
For security reasons, further NPM releases will be done manually. This way we don't have to add a static token here and can use 2FA locally.

Also added branch protection rule.

### CODE REVIEW

File: .github/workflows/linter.yml [hunk 1]
```diff
@@ -1,19 +1,46 @@
...
+  NODE_VERSION: 18
+  VALIDATE_ALL_CODEBASE: true
+  YAML_CONFIG_FILE: /.github/linters/.yamllint.yaml
 
 jobs:
   build:
     name: Lint Code Base
     runs-on: ubuntu-latest
-
+    timeout-minutes: 70
```
Comment (rafaelmaeuer) [inline .github/workflows/linter.yml:24]: very generous don't you think?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+- [SWR Prettier config](#swr-prettier-config)
+  - [Installation](#installation)
+  - [Editor Config](#editor-config)
+
+## Installation
+
+Step 1: Install [npm package](https://www.npmjs.com/package/@swrlab/swr-prettier-config)
+
+```shell
+npm install @swrlab/swr-prettier-config
```
Comment (rafaelmaeuer) [inline README.md]:
```suggestion
npm install @swrlab/swr-prettier-config --save-dev
```

File: package.json [hunk 3]
```diff
@@ -1,22 +1,26 @@
 {
 	"name": "@swrlab/swr-prettier-config",
-	"version": "0.1.2",
-	"description": "SWR Prettier config.",
+	"version": "0.2.0",
```
Comment (rafaelmaeuer) [inline package.json:3]: how about adding a changelog file?

File: package.json [hunk 4]
```diff
@@ -1,22 +1,26 @@
...
+	"description": "SWR Audio Lab Prettier config",
+	"license": "MIT",
```
Comment (rafaelmaeuer) [inline package.json:5]: we need to mirror the new license in ard-eventhub readme

**Review by rafaelmaeuer — CHANGES_REQUESTED** [review]

File: .github/workflows/linter.yml (hunk 1 above)
Comment (frytg) [reply .github/workflows/linter.yml:24]: Radiohub currently needs ~60min and this way we can keep the same file everywhere

File: package.json (hunk 4 above)
Comment (frytg) [reply package.json:5]: good idea https://github.com/swrlab/ard-eventhub/issues/373

File: package.json (hunk 3 above)
Comment (frytg) [reply package.json:3]: Will add in next version

**Pull request merged**